tests/integration/targets/ios_banner/tests/cli/net_banner.yml

```diff
@@ -1,29 +0,0 @@
----
```

Comment:
Why are we removing net_banner tests?

Reply:
https://github.com/ansible-collections/ansible.netcommon/blob/dde94450cc9cc6d07a52b98c1eec8a1fcfc5240c/plugins/modules/net_banner.py#L13
it's deprecated and tobe removed from netcommon in the upcoming release